shaders/Monochrome.shader

```diff
@@ -0,0 +1,9 @@
+shader_type canvas_item;
+
+uniform vec3 rgbMultipliers = vec3(1, 1, 1);
```

Comment:
Aren't these values wrong per the earlier discussion on this file?

Reply:
What exactly do you mean? Monochrome works by calculating a weighted average of the rgb values and setting all color vectors to that value. This variable defines the weights for the three color vectors. By default all channels should be treated equaliy.

Reply:
Umm, no? Didn't you see those numbers posted here: https://github.com/Revolutionary-Games/Thrive/pull/2623/files/089fc3743df646ce7a68489651d837d4b8c66749#r739664387 ? Each RGB value should have a different default contribution factor to the final luminosity.